packages/amplication-code-gen-types/src/code-gen-types.ts

```diff
@@ -315,3 +325,16 @@ export type BuildContext = {
   projectId: string;
   data: DSGResourceData;
 };
+
+export type EntityComponent = {
+  name: string;
+  file: namedTypes.File;
+  modulePath: string;
+};
+
+export type EntityComponents = {
+  new: EntityComponent;
+  list: EntityComponent;
+  edit: EntityComponent;
+  show: EntityComponent;
+};
```

Comment:
@Shurtu-gal these types are already defined in `packages\data-service-generator\src\admin\types.ts` so if you moved them here you should remove them from there